client/src/features/live-site/constants/resources.tsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable react/no-unescaped-entities */
+import Link from "next/link";
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "~/components/ui/accordion";
+
+
+export function FreeResources() {
+  return (
+    <div className="grid grid-cols-1 gap-8">
+      <Accordion type="multiple" className="w-full">
+        <AccordionItem value="Starter Kits">
+          <AccordionTrigger value={"1"}>
+            <h2 className="text-xl text-left font-semibold">Starter Kits</h2>
+          </AccordionTrigger>
+          <AccordionContent className="bg-muted">
+            <h2 className="text-md font-large">Build Your Own</h2>
+            <div className="underline text-primary">
+              <br />
```

Comment:
Similarly, you could probably use `ul` and `li` and not have these `br` tags. Alternatively, you could use a column flex box.